### DISCUSSION TITLE
Request to open links in a new tab

### DISCUSSION THREAD

**Discussion by debbie-oyek** [GitbookIO/community#679] [Feature Requests]
Requesting to have the option to open/view links (internal and external) in a new tab instead of opening it in the current tab, which replaces the entire page. Sometimes when I create a cross-ref link to somewhere within the document itself, having it open to a new tab is helpful (to check let's say that I linked to the correct one). Thanks!

**Comment by aikoii** [comment]
Hey @debbie-oyek! We have a similar suggestion here #577, I encourage you to vote for it 🙂.

We don't have any immediate plans to support this yet, but appreciate you sharing your thoughts!

(I will be closing this in favor of the first request)
